src/extensions/flatJSON/bootstrap-table-flatJSON.js

```diff
@@ -27,8 +27,8 @@
         _initTable = BootstrapTable.prototype.initTable;
 
     BootstrapTable.prototype.initData = function (data) {
-        _initData.apply(this, Array.prototype.slice.apply(arguments));
         flatJSON(this, data === undefined ? this.options.data : data);
+        _initData.apply(this, Array.prototype.slice.apply(arguments));
```

Comment:
I'm not sure about this change.. Did you test it with this change and without it?